ci-operator/step-registry/hypershift/kubevirt/conformance/hypershift-kubevirt-conformance-workflow.yaml

```diff
@@ -24,6 +24,7 @@ workflow:
     pre:
     - chain: ipi-aws-ovn-pre
     - chain: ipi-install
+    - ref: hypershift-kubevirt-install-odf
```

Comment:
for now, lets leave ODF out of the conformance tests.

The reason being, these conformance tests are used across multiple OCP versions, and not just the latest ocp. I want to make sure we keep the conformance tests stable, so we likely need to take more into consideration before introducing ODF here. 

We'll definitely want to follow up on this though. I just want to limit the blast radius of this initial change.

Reply:
Done